util/__init__.py

```diff
@@ -173,6 +175,7 @@ def setup(args, extra):
     print('Setting up the Meerkat codebase...')
     print('This will destroy changes, resetting everything to the remote.')
 
+    create_dummy_aws_config()
```

Comment:
This line should probably be after checking with the user whether they want to continue. If the user says no, we should probably not do anything at all. Maybe just move it down two lines?